src/__generated__/ArtistArtworks_artworks.graphql.ts

```diff
@@ -0,0 +1,88 @@
+/* tslint:disable */
+
+import { ConcreteFragment } from "relay-runtime";
+export type ArtistArtworks_artworks = ReadonlyArray<{
+    readonly artist: ({
```

Comment:
@alloy / @orta - what was that trick to automatically collapsing these `__generated__` files in github reviews?

Reply:
`.gitattributes` - https://github.com/artsy/reaction/blob/master/.gitattributes